test/input/drv_clmin.dat.old

```diff
@@ -0,0 +1,220 @@
+!=========================================================================                                                                                                                                                                                

```

Comment:
@smithsg84 Garrett had added this file and was using it for tests. We are still using it for tests. I'm not sure what makes it "old".

@grapp1

Reply:
That file is used for some validation in one of our test. I guess it was name old rather than origin since now, we can generate it from some parflow properties using py-pftools.